mender-update/daemon/states.cpp

```diff
@@ -70,7 +70,9 @@ void StateScriptState::OnEnter(Context &ctx, sm::EventPoster<StateEvent> &poster
 	string state_name {script_executor::Name(this->state_, this->action_)};
 	log::Debug("Executing the  " + state_name + " State Scripts...");
 	auto err = this->script_.AsyncRunScripts(
-		this->state_, this->action_, [state_name, &poster](error::Error err) {
+		this->state_,
+		this->action_,
+		[state_name, &poster](error::Error err) {
```

Comment:
The only exception is this.

Btw, do you know how to run clang-format individually on each commit? Ping me if you want me to show you.

Reply:
@kacf ping! I'd like to learn your process.

Reply:
For future reference, check out https://gitlab.com/kacf/gems.